doc/nrf/gs_installing.rst

```diff
@@ -637,6 +637,19 @@ Define the required environment variables as follows, depending on your operatin
       This file is loaded automatically when you run the above command.
       See :ref:`zephyr:env_vars_zephyrrc` for more information.
 
+Install nrfjprog
+================
+
 You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.
 The west command programs the development kit by using nrfjprog by default.
```

Comment:
```suggestion
The west command uses nrfjprog to program the development kit by default.
```
I think that would be a more readable sentence.

Reply:
Thanks! I added this change.